libraries/src/test/java/com/baeldung/infinispan/AbstractIntegrationTest.java

```diff
@@ -8,10 +8,12 @@
 import org.infinispan.manager.DefaultCacheManager;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
 
 import java.util.function.Supplier;
 
-public class ConfigurationTest {
+@Ignore
+public abstract class AbstractIntegrationTest {
```

Comment:
@pivovarit important change, this class is used by two further integration tests, this test class only provides common setup and tear down logic to its child test classes, hence this has been named abstract and marked ignored.